poster_data/auto-load-with-log.sh

```
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Create log file with timestamp
+LOG_FILE="poster_data/auto-load-$(date +%Y%m%d-%H%M%S).log"
+
+echo "Starting auto-load process at $(date)" | tee "$LOG_FILE"
+echo "Log file: $LOG_FILE" | tee -a "$LOG_FILE"
+echo "Arguments: $@" | tee -a "$LOG_FILE"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)